src/main/kotlin/com/github/pemistahl/lingua/api/LanguageDetectorBuilder.kt

```diff
@@ -58,6 +64,22 @@ class LanguageDetectorBuilder private constructor(
         return this
     }
 
+    /**
+     * Configures the language detector to not use quadrigram and fivegram language models for
+     * language detection. This affects both dynamically loaded models as well as
+     * [preloaded models][withPreloadedLanguageModels].
```

Comment:
Maybe the wording "dynamically loaded modeles as well as preloaded models" is a bit misleading. It might not be clear enough what this means by "dynamically loaded", and it might sound as if even a detector with preloaded models could dynamically load models.

Any suggestions for alternative wordings, or is this sentence fine?